### PR TITLE
KB-8777: Very small changes on the client following number of usage

### PR DESCRIPTION
**Pull request by amedeepotier**
I would also like to add Sasha... but I do not see him in the list of members. 
This is a very small change there..

### CODE REVIEW

File: openchatbotclient/response.py [hunk 1]
```diff
@@ -92,3 +93,65 @@ def is_success(self):
...
+    # Set of utility properties to grab the most typical data
+    # from the json
+    #
+
+    #
+    # Related to the query:
+    #
+    @property
+    def query(self) -> str:
+        return self.jso.get('response', {}).get('query', '')
```
Comment (andreyjamer) [inline openchatbotclient/response.py:106]: Why the property is named as `jso`? Is it for JSON? Or JSON Storable Object? I think it should be renamed into something simple and clear.

File: openchatbotclient/response.py [hunk 2]
```diff
@@ -84,6 +84,7 @@ def __str__(self):
         return 'response(%s => %s)' % (self.cli, self.get_text())
 
     def get_client(self):
+        """Returns an instance of 'client' class"""
         return self.cli
```
Comment (andreyjamer) [inline openchatbotclient/response.py:88]: Why client is `cli`? CLI - is an acronym for Command Line Interface. I think this property should be renamed into `client`

File: openchatbotclient/response.py [hunk 3]
```diff
@@ -92,3 +93,65 @@ def is_success(self):
...
+    def userId(self) -> str:
+        return self.jso.get('response', {}).get('userId', '')
+
+    #
+    # Response data
+    #
+    @property
+    def code(self) -> int:
+        """The HTTP response code, such as 200"""
+        return self.jso.get('status', {}).get('code', '')
```
Comment (andreyjamer) [inline openchatbotclient/response.py:118]: Result value is announced as `int`, but in get('code') default value is string. This is not good

**Review by andreyjamer — CHANGES_REQUESTED** [review]
Main questions are about code style. properties names are not obvious enough.
If we are building open source solution, then we should follow common style guides, like PEP8. For example class names should use the CapWords convention (https://www.python.org/dev/peps/pep-0008/#class-names). Current implementation does not use it.

File: openchatbotclient/response.py (hunk 1 above)
Comment (amedeepotier) [reply openchatbotclient/response.py:106]: Understood your comment, but your note about invalid class name is related on this pull request !! Your note is about the original work itself. I am happy to rewrite the package to be compliant with the PIP recommendations... but I will not do this under this ticket, which is only about minor enhancements on the new properties). 

The new properties added in this PR are inline with the python recommendations.

File: openchatbotclient/response.py (hunk 2 above)
Comment (amedeepotier) [reply openchatbotclient/response.py:88]: Again, this is about previous code... not related to this PR. But yes, I do agree with you. Will make this change (but not on this PR, too late in the sprint to rewrite the package now)

File: openchatbotclient/response.py (hunk 3 above)
Comment (amedeepotier) [reply openchatbotclient/response.py:118]: yes, good point. Will solve this shortly

**Review by andreyjamer — APPROVED** [review]
Approved

**Comment by amedeepotier** [comment]
Merging to master

**Pull request merged**